kubeadm/v1beta1-config.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: kubeadm.k8s.io/v1beta1
+kind: InitConfiguration
+localAPIEndpoint:
+  advertiseAddress: CONFIG_CLUSTER_PRIVATE_IP
+  bindPort: 6443
+nodeRegistration:
+  kubeletExtraArgs:
+    "feature-gates": "BlockVolume=true,CRIContainerLogRotation=true"
```

Comment:
Having the featuregates as TF variables would be good too.
Just not sure how to make that variable

"BlockVolume=true,CRIContainerLogRotation=true"

featureGates:
  BlockVolume: true
  CRIContainerLogRotation: true

Some parsing logic would be needed to convert to yaml.

I suppose we could just add it later or have you any ideas?

Reply:
I would leave it for later.